extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -622,7 +635,7 @@ function processFilePickerField(context: FieldContext): FilePickerInputs {
 			defaultUri: vscode.Uri.file(path.dirname(input.value || os_homedir())),
 			openLabel: loc.select,
 			filters: {
-				'Config Files': ['*'],
+				'File': ['*'],
```

Comment:
Localize?

Reply:
I will expose this as a property of file_picker field, localization will be handled by extension.

Reply:
**filter not specified**
![Screen Shot 2020-07-06 at 7 43 31 PM](https://user-images.githubusercontent.com/13777222/86695322-48d91100-bfc1-11ea-9c9b-366f3bd847e4.png)

**filter specified**
![Screen Shot 2020-07-06 at 7 43 51 PM](https://user-images.githubusercontent.com/13777222/86695329-4aa2d480-bfc1-11ea-9120-61a088ea1261.png)